docs/blog/2019-05-21-setting-up-yarn-workspaces-for-theme-development/index.md

```diff
@@ -0,0 +1,213 @@
+```
+
+```js:title=gatsby-theme-example-workspaces/gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: "gatsby-plugin-compile-es6-packages",
+      options: {
+        modules: ["gatsby-theme-example-workspaces"],
+      },
+    },
+  ],
+}
+```
+
+Change the `main` field in your theme's `package.json` to point to the `gatsby-config.js` file.
+
+```json:title=gatsby-theme-example-workspaces/package.json
+{
+  "name": "gatsby-theme-example-workspaces",
+  "description": "",
+  "version": "1.0.0",
+  "main": "gatsby-config.js", // highlight-line
+  "license": "MIT",
+  "peerDependencies": {
+    "gatsby": "^2.2.10",
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  },
+  "dependencies": {
+    "gatsby-plugin-compile-es6-packages": "^1.0.6"
+  }
+}
+```
+
+## Example site setup
+
+Add the theme as a dependency to the example site.
+By specifying the version here, Yarn will install the local dependency that hasn't been published to npm yet.
+
+```shell
+yarn workspace example add gatsby-theme-example-workspaces@1.0.0
+```
+
+In the example site, create a `gatsby-config.js` file and add the theme.
+
+```js:title=example/gatsby-config.js
+module.exports = {
+  __experimentalThemes: ["gatsby-theme-example-workspaces"],
+}
+```
+
+Add a `src/pages/` directory and add a _Hello, world_ page.
+
+```js:title=example/src/pages/index.js
+import React from "react"
+
+export default props => <h1>Hello, world</h1>
+```
+
+Add Gatsby develop and build scripts to the example site's `package.json`.
+
+```json:title=example/package.json
+{
+  "name": "example",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  // highlight-start
+  "scripts": {
+    "develop": "gatsby develop",
+    "build": "gatsby build"
+  },
+  // highlight-end
+  "dependencies": {
+    "gatsby": "^2.2.10",
+    "gatsby-theme-example-workspaces": "1.0.0",
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  }
+}
+```
+
+Test your example site out to make sure everything is working as expected.
+
+```shell
+yarn workspace example develop
+```
+
+Open your browser to `http://localhost:8000` where you should see your _Hello, world_ page.
+
+## Adding functionality to the theme
+
+Currently, the theme does nothing.
+This is the minimal amount of code required to develop a theme with Yarn workspaces.
+
+To demonstrate the theme in action, use `gatsby-plugin-page-creator` to add a page from the theme.
+
+```shell
+yarn workspace gatsby-theme-example-workspaces add gatsby-plugin-page-creator
+```
+
+```js:title=gatsby-theme-example-workspaces/gatsby-config.js
+const path = require("path") // highlight-line
+module.exports = {
+  plugins: [
+    // highlight-start
+    {
+      resolve: "gatsby-plugin-page-creator",
+      options: {
+        path: path.join(__dirname, "src/pages"),
+      },
+    },
+    // highlight-end
+    {
+      resolve: "gatsby-plugin-compile-es6-packages",
```

Comment:
If I remember correctly, this is no longer required, right? I think @ChristopherBiscardi put in a PR that removed the need in #13651